lib/utils.js

```diff
@@ -50,7 +50,13 @@ async function resolveExecutablePath (cmd) {
       }
     }
   } catch (err) {
-    log.warn(err);
+    if (err.code === 1) {
```

Comment:
I was thinking about using `fs.which` instead, which implements the same functionality